src/server/GameController.cpp

```diff
@@ -29,16 +28,12 @@ void GameController::update() {
 }
 
 client_inputs GameController::collectInputs() {
-    // Give clients a window to write inputs
-    mNetworkingServer->collectClientInputs();
-
-    // Read client inputs
-    return mNetworkingServer->getClientInputs();
+    return mNetworkingServer->collectClientInputs();
 }
 
 void GameController::computeGameState(
-    std::vector<int> client_ids,
-    std::vector<client_direction_update> client_direction_updates
+    std::vector<int>& client_ids,
```

Comment:
why change these to be references

Reply:
Code-climate (cppcheck) pointed out: "Parameter client_ids is passed by value. It could be passed as a const reference which is usually faster and recommended in C++."

Basically doesn't need to copy the whole vector (which is faster).